pkgs/by-name/mo/modrinth-app-unwrapped/package.nix

```diff
@@ -169,25 +112,20 @@ rustPlatform.buildRustPackage {
       runHook postInstall
     '';
 
-  passthru = {
-    updateScript = nix-update-script { };
-  };
-
   meta = {
     description = "Modrinth's game launcher";
     longDescription = ''
       A unique, open source launcher that allows you to play your favorite mods,
       and keep them up to date, all in one neat little package
     '';
-    mainProgram = "modrinth-app";
     homepage = "https://modrinth.com";
-    changelog = "https://github.com/modrinth/theseus/releases/tag/v${modrinth-app-unwrapped.version}";
     license = with lib.licenses; [
       gpl3Plus
       unfreeRedistributable
     ];
     maintainers = with lib.maintainers; [ getchoo ];
```

Comment:
```suggestion
    license = [
      licenses.gpl3Plus
      licenses.unfreeRedistributable
    ];
    maintainers = [ maintainers.getchoo ];
```

Reply:
Limiting use of `with` to very small scopes like lists is a generally accepted compromise.